bin/MadGraph5_aMCatNLO/cards/production/13TeV/3Top/TTTJMinus/TTTJMinus_NLO_5FS_DR1_run_card.dat

```diff
@@ -0,0 +1,195 @@
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1	= run_tag ! name of the run 
+#***********************************************************************
+# Number of LHE events (and their normalization) and the required      *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+  10000	= nevents ! Number of unweighted events requested 
+  -1.0	= req_acc ! Required accuracy (-1=auto determined from nevents)
+  -1	= nevt_job ! Max number of events per job in event generation. 
+                 !  (-1= no split).
+#***********************************************************************
+# Output format
+#***********************************************************************
+  -1.0	= time_of_flight ! threshold (in mm) below which the invariant livetime is not written (-1 means not written)
+  average	= event_norm ! average/sum/bias. Normalization of the weight in the LHEF
+#***********************************************************************
+# Number of points per itegration channel (ignored for aMC@NLO runs)   *
+#***********************************************************************
+  0.01	= req_acc_fo ! Required accuracy (-1=ignored, and use the 
+ 	                   ! number of points and iter. below)
+# These numbers are ignored except if req_acc_FO is equal to -1
+  5000	= npoints_fo_grid ! number of points to setup grids
+  4	= niters_fo_grid ! number of iter. to setup grids
+  10000	= npoints_fo ! number of points to compute Xsec
+  6	= niters_fo ! number of iter. to compute Xsec
+#***********************************************************************
+# Random number seed                                                   *
+#***********************************************************************
+  0	= iseed ! rnd seed (0=assigned automatically=default))
+#***********************************************************************
+# Collider type and energy                                             *
+#***********************************************************************
+  1	= lpp1 ! beam 1 type (0 = no PDF)
+  1	= lpp2 ! beam 2 type (0 = no PDF)
+  6500.0	= ebeam1 ! beam 1 energy in GeV
+  6500.0	= ebeam2 ! beam 2 energy in GeV
+#***********************************************************************
+# PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
+#***********************************************************************
+  lhapdf	= pdlabel ! PDF set
+  244600	= lhaid ! If pdlabel=lhapdf, this is the lhapdf number. Only 
+              ! numbers for central PDF sets are allowed. Can be a list; 
+              ! PDF sets beyond the first are included via reweighting.
```

Comment:
change them to:
   'lhapdf'    = pdlabel     ! PDF set                                  
    $DEFAULT_PDF_SETS = lhaid
    $DEFAULT_PDF_MEMBERS = reweight_PDF     ! if pdlabel=lhapdf, this is the lhapdf number